odoo/addons/base/ir/ir_qweb/fields.py

```diff
@@ -167,21 +167,11 @@ class DateConverter(models.AbstractModel):
 
     @api.model
     def value_to_html(self, value, options):
-        if not value or len(value) < 10:
-            return ''
-        lang = self.user_lang()
-        locale = babel.Locale.parse(lang.code)
-
-        if isinstance(value, basestring):
-            value = fields.Datetime.from_string(value[:10])
-
+        date_format = False
         if options and 'format' in options:
-            pattern = options['format']
-        else:
-            strftime_pattern = lang.date_format
-            pattern = posix_to_ldml(strftime_pattern, locale=locale)
+            date_format = options['format']
```

Comment:
I missed that part, but here are some calls where it is `None`: https://github.com/odoo-dev/enterprise/blob/master-ref_format_lang_date-nim/account_reports/models/account_report.py#L376-L394
(I will correct it in the corresponding enterprise PR)